### PR TITLE
Fix/repex support

### PR DESCRIPTION
**Pull request by andre-merzky**
This PR adds two attributes to EnTK tasks: `task.link_output_data` and `task.sandbox`.  I am not sure why output linking was missing, as the linking for input data exists.  The sandbox supports simplifies tracking of data for the repex use case, and simply maps to the RP task attribute of the same name.

### CODE REVIEW

**Comment by lee212** [comment]
Ping @andre-merzky

**Comment by andre-merzky** [comment]
Hey @lee212 : I am sorry, what's the ping about specifically?

**Comment by lee212** [comment]
All looks good to me except the line 360 at src/radical/entk/execman/rp/task_processor.py
 with:
```
    'action': rp.COPY
```

Isn't it `rp.LINK`?

File: src/radical/entk/execman/rp/task_processor.py [hunk 1]
```diff
@@ -341,6 +341,26 @@ def get_output_list_from_task(task, placeholders):
...
+                    temp = {
+                        'source': path.split('>')[0].strip(),
+                        'target': path.split('>')[1].strip(),
+                        'action': rp.LINK
+                    }
+                else:
+                    temp = {
+                        'source': path.split('>')[0].strip(),
+                        'target': os.path.basename(path.split('>')[0].strip()),
+                        'action': rp.COPY
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/entk/execman/rp/task_processor.py]: Ah, good catch!  Thanks, will fix.

**Comment by andre-merzky** [comment]
Thanks for noting that, the flag is fixed to `LINK` now

**Comment by andre-merzky** [comment]
Oh, I can't recollect either.  From the topic of the PR I would actually think that the sleep does not belong here at all.  Do you happen to know if the code still behaves with the sleep removed?

**Pull request merged**